### PR TITLE
Feature/liayoo/sandbox expr

### PR DESCRIPTION
**Pull request by liayoo**
Decrease max_num_validators for sandbox: 30 -> 20

### CODE REVIEW

**Review by minsulee2 — APPROVED** [review]
LGTM!

File: blockchain-configs/testnet-sandbox/blockchain_params.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "min_stake_for_proposer": 10000000,
     "max_stake_for_proposer": 10000000,
     "min_num_validators": 5,
-    "max_num_validators": 30,
+    "max_num_validators": 20,
```
Comment (platfowner) [inline blockchain-configs/testnet-sandbox/blockchain_params.json:27]: what about applying this to other testnet clusters (dev, staging, prod) as well?

Comment (liayoo) [reply blockchain-configs/testnet-sandbox/blockchain_params.json:27]: done!!

**Review by platfowner — APPROVED** [review]
LGTM
Thanks!

**Comment by liayoo** [comment]
Thanks for the fast reviews! Now merging..

**Pull request merged**